src/main/java/com/github/seikichi/pronto_checkstyle_findbugs_example/App.java

```diff
@@ -6,6 +6,9 @@ public String getGreeting() {
   }
 
   public static void main(String[] args) {
+    if ("foo" == String.format("%soo", "f")) { // findbugs violation
+    System.out.println("foo!"); // checkstyle violation
```

Comment:
checkstyle &apos;if&apos; child has incorrect indentation level 4, expected level should be 6.